### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by OlehLapovets** [flymia/SpotCurrent#9]
Do you still support this app?
It generally starts, but shows that song is paused
I'm using ubuntu 22.04 LTS

**Comment by flymia** [comment]
Hi there! Thanks for opening up an issue. I will have a look into it.

**Comment by bailboy91** [comment]
I am currently playing around and noticed the Spotify app no longer adds anything to the title of the app.  so wmcrtl -l -p only Displays "Spotify" and not "Spotify - Song title / artist"

`0x06c00003  0 6      zenmint Spotify`

I am still going to mess around and see if there's an option anywhere.

**Comment by flymia** [comment]
You are right. Seems like they removed the option. The only way to get the current song playing may be to use something like dbus, since the desktop environment (KDE, GNOME, etc.) get the current song playing, also.
